src/gettingstarted/cli.md

```diff
@@ -97,3 +97,30 @@ Once Bash for Windows is installed, you can install the Platform.sh CLI with the
 ```bash
 curl -sS https://platform.sh/cli/installer | php
 ```
+
+## API Tokens
+
+The CLI can also be used from 3rd party CI services or other automation tools, and supports an API Token authentication option for that purpose.
+
+An API token can be created through the UI.  Go to the "User" page from your accounts drop-down, then select the "Account Settings" tab, then the "API Tokens".
```

Comment:
Is "your accounts drop-down" clear?

Reply:
I'm not sure.  I didn't want to go through and add a dozen extra screenshots, but text is hard. :smile: 

@josephdpurcell Your thoughts?  Was this sufficiently clear?

Reply:
It is sufficiently clear after having gone to the "User" page before. I admit when I very first logged into Platform.sh and clicked my profile image drop down and didn't see a page that said "My Profile" or "Settings" or something similar, I was confused. I thought the simple word "User" was ambiguous, oddly enough.

All that said, I think the way this is written is fine. People who get to this page will most likely have already been to the "User" page before and will know what the phrase `Go to the "User" page from your accounts drop-down` means.